### PR TITLE
Create a build process using webpack

### PR DESCRIPTION
**Pull request by bnjmnhndrsn**
Creates a build process using webpack and babel.

`npm run start` starts a webpack dev server with hot reloading

`npm run build:production` builds the production assets.

As we'll be serving the assets from Github for the time being, we'll have to check in the built scripts to the repo.

### CODE REVIEW

File: dist/bundle.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
...tentLoaded",function(e){var n=function(e){for(var n={},t=("?"===e[0]?e.substr(1):e).split("&"),r=0;r<t.length;r++){var o=t[r].split("=");n[decodeURIComponent(o[0])]=decodeURIComponent(o[1]||"")}return n}(document.location.search);!function(e,n){var t=document.body,r=document.createElement("audio");r.controls=!0,r.autoplay=!0,r.src=e,t.appendChild(r);var o=function(e){r.currentTime=n,r.play(),r.removeEventListener("canplay",o)};r.addEventListener("canplay",o)}(decodeURIComponent(n.src),n.time)})},"./src/styles.css":function(e,n,t){var r=t("./node_modules/css-loader/index.js!./src/styles.css");"string"==typeof r&&(r=[[e.i,r,""]]);var o={hmr:!0,transform:void 0,insertInto:void 0},i=t("./node_modules/style-loader/lib/addStyles.js")(r,o);r.locals&&(e.exports=r.locals),e.hot.accept("./node_modules/css-loader/index.js!./src/styles.css",function(n){!function(){var n=t("./node_modules/css-loader/index.js!./src/styles.css");if("string"==typeof n&&(n=[[e.i,n,""]]),!function(e,n){var t,r=0;for(t in e){if(!n||e[t]!==n[t])return!1;r++}for(t in n)r--;return 0===r}(r.locals,n.locals))throw new Error("Aborting CSS HMR due to changed css-modules locals.");i(n)}()}),e.hot.dispose(function(){i()})}});
```
Comment (psynautic) [inline dist/bundle.js:1]: should we gitignore this?

File: dist/index.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```
Comment (psynautic) [inline dist/index.html:1]: just all of dist/*

Comment (bnjmnhndrsn) [reply dist/index.html:1]: So the way we have it set up, we're serving the page from Github. So we need this checked in. Two other potential options:
 - we serve the compiled script from somewhere else (e.g. Heroku)
 - we set up github pages configuration so it's serving a branch other than master and then have the deploy process work like this:
   - merge into master
   - checkout other branch and merge in master
   - build and the assets and push

Comment (psynautic) [reply dist/index.html:1]: oh right. eh the second process feels janky. The idea of having a 'prod' and 'master' branch feel off.

Comment (bnjmnhndrsn) [reply dist/index.html:1]: yeah. I say we go with checking in the scripts for now and then decide to deploy elsewhere if we actually keep using this.

**Review by psynautic — APPROVED** [review]
🐿️🐿️🐿️ it

**Pull request merged**